src/components/NewCard/README.md

```diff
@@ -0,0 +1,41 @@
+## NewCard
+An implementation of an NewCard for TPAs
+
```

Comment:
We should probably add that this is the formal `Card` that should be used, and that the former `Card` component will be deprecated.